packages/ui-components/src/components/hero-component.ts

```diff
@@ -123,8 +123,12 @@ export class HeroComponent extends BaseElement {
               See your reports from anywhere
             </p>
             <div class="flex justify-center space-x-4 my-8">
-              <sme-button @click="${this.#handlePrimary}">Get started</sme-button>
-              <sme-button @click="${this.#handleSecondary}" priority="secondary">What is Stryker?</sme-button>
+              <sme-button @click="${this.#handlePrimary}"
+                >Get started</sme-button
+              >
```

Comment:
Not necessarily for this PR, but I really don't like this formatting... 🙃

Reply:
I guess increasing the width to something higher than 80 (?) would probably suffice, but we can do that later (in the actual repository).